core/src/avm2/e4x.rs

```diff
@@ -475,36 +490,58 @@ impl<'gc> E4XNode<'gc> {
     /// valid encoded UTF-8 data. (Other encoding support is planned later.)
     pub fn from_start_event(
         activation: &mut Activation<'_, 'gc>,
+        parser: &NsReader<&[u8]>,
         bs: &BytesStart<'_>,
         decoder: quick_xml::Decoder,
     ) -> Result<Self, quick_xml::Error> {
-        // FIXME - handle namespace
-        let name =
-            AvmString::new_utf8_bytes(activation.context.gc_context, bs.local_name().into_inner());
-
         let mut attribute_nodes = Vec::new();
 
         let attributes: Result<Vec<_>, _> = bs.attributes().collect();
         for attribute in attributes? {
-            let key = AvmString::new_utf8_bytes(
-                activation.context.gc_context,
-                attribute.key.into_inner(),
-            );
+            let (ns, local_name) = parser.resolve_attribute(attribute.key);
+            let namespace = match ns {
+                ResolveResult::Bound(ns) => Some(AvmString::new_utf8_bytes(
+                    activation.context.gc_context,
+                    ns.into_inner(),
+                )),
+                ResolveResult::Unknown(ns) if ns == b"xmlns" => continue,
+                // TODO: Throw error on unknown prefix
```

Comment:
Can you add an `avm2_stub_method!` call here?

Reply:
I just went ahead and added the correct errors in the second commit.